README.md

```diff
@@ -25,6 +25,30 @@ Builds the app for production in the `build` folder.
 `yarn eject`
 Ejects the app from `create-react-app` rails.
 
+## Testing
+
+### Performing E2E Tests with Cypress
+
+To run e2e test with Cypress, first you must serve a local instance
```

Comment:
Note: Once this is approved and integrated, this can be pointed to a staging/prod environment

Reply:
Or, this could be ignored and this can be part of CI